src/components/Table/Table.jsx

```diff
@@ -108,6 +108,9 @@ class Table extends React.Component {
       currentPage: this.state.currentPage - 1,
     });
   }
+  getURLParameter(name) {
```

Comment:
I suggest adding a filter option to odota/core called "having" or something (the SQL equivalent to this filter), which ties in better with the existing filter framework